stdlib/public/TensorFlow/DataTypes.swift

```diff
@@ -21,11 +21,17 @@
 
 import CTensorFlow
 
+/// Wraps a TF_DataType.
```

Comment:
This is public API while `TF_DataType` is completely internal. I'd rather drop the description that mentions `TF_DataType` and write something that users can internalize.

I suggest:
```swift
/// A TensorFlow dynamic type value. Can be created from types that conform to `AccelerableByTensorFlow`.
```

Then you can write whatever internal documentation below starting with _two_ slashes.